controller/src/controller/config.py

```diff
@@ -498,6 +499,123 @@ class EnabledFileserverConfig(FileserverConfig):
+        ),
+    ] = None
+
+    application: Annotated[
+        str | None,
+        Field(
+            title="Argo CD application",
+            description="Argo CD application into which to put fsadmin pods",
+        ),
+    ] = None
+
+    extra_annotations: Annotated[
+        dict[str, str],
+        Field(
+            title="Extra annotations",
+            description=(
+                "Extra annotations to add to all user file server ``Job`` and"
+                " ``Pod`` Kubernetes resources"
+            ),
+        ),
+    ] = {}
+
+    # We default this so we don't need a configuration-file flag day synced
+    # to the nublado updates.
+    image: Annotated[
+        ContainerImage,
+        Field(
+            title="fsadmin Docker image",
+            description="Docker image to run as fsadmin",
+        ),
+    ] = ContainerImage(
+        repository="ghcr.io/lsst-sqre/nublado-fsadmin",
+        pull_policy=PullPolicy.ALWAYS,
+        tag="latest",
+    )
+
+    mount_prefix: Annotated[
+        str | None,
+        Field(
+            title="Prefix for fsadmin mounts",
+            description=(
+                "If given, the mounts will be collected under this directory."
+            ),
+            examples=["/mnt"],
+        ),
+    ] = None
+
+    node_selector: Annotated[
+        dict[str, str],
+        Field(
+            title="File server node selector",
+            description=(
+                "Labels that must be present on Kubernetes nodes for any"
+                " fsadmin pods to be scheduled there"
+            ),
+            examples=[{"disktype": "ssd"}],
+        ),
+    ] = {}
+
+    pod_name: Annotated[
+        str,
+        Field(
+            title="fsadmin pod name",
+            description="Pod/container name for fsadmin",
+        ),
+    ] = "fsadmin"
+
+    resources: Annotated[
+        LabResources | None,
+        Field(
+            title="Resource requests and limits",
+            description=(
+                "Kubernetes resource requests and limits for fsadmin pods"
+            ),
+        ),
+    ] = None
+
+    timeout: Annotated[
+        HumanTimedelta,
+        Field(
+            title="Creation timeout",
+            description=(
+                "How long to allow for fsadmin pod to be created/deleted"
+            ),
+        ),
+    ] = timedelta(minutes=2)
+
+    tolerations: Annotated[
+        list[Toleration],
+        Field(
+            title="File server pod tolerations",
+            description="Kubernetes tolerations for fsadmin pods",
+        ),
+    ] = []
+
+    volume_mounts: Annotated[
+        list[VolumeMountConfig],
+        Field(
+            title="Volume mounts",
+            description=(
+                "Volumes mounted in the fsadmin pod. The ``containerPath``"
+                " settings represent the path visible inside the container."
+            ),
+        ),
+    ] = []
```

Comment:
Shouldn't this just be copied from labs? Why would we want to mount the volumes in different places in the fsadmin container than in the user labs? Or are there other settings that would need to be different for the admin case? I don't really like having to duplicate this (and make sure it refers correctly to the lab volumes).

If you do keep this, it needs a validator to make sure that it's consistent with `lab.volumes`.